src/opcodes.js

```diff
@@ -6,7 +6,7 @@ const wordInfoMap = require('../data/word_info_map')
 function opcodeForWord (word) {
   assert(wordIsValid(word), `${word} is not a valid term`)
   const info = wordInfoMap[word]
-  const idx = info.words.indexOf(word)
+  const idx = info.opcodes.length === 1 ? 0 : info.words.indexOf(word)
```

Comment:
Does this work in all cases? Are there any cases where there are more than 1 opcode, but more than 2 words?

Reply:
@JBaczuk I looked over the [opcodes.json](https://github.com/JBaczuk/bscript-parser/blob/master/data/opcode_info.json) file to verify if this works in all cases and it does.  There are four ways the opcodes map to their ASM terms.  The first, and by far the most common, is one-to-one, where there is a single opcode mapping to a single ASM term.  The second, which is a special case, is many-to-zero, which exists for all push_data(n) opcodes (`0x01`-`0x4b`).  The third is one-to-many, where there is a single opcode mapping to multiple ASM terms.  There are a few cases of this in the data file: `OP_0`/`OP_FALSE`, `OP_1`/`OP_TRUE`, `OP_CHECKLOCKTIMEVERIFY`/`OP_NOP2` and `OP_CHECKSEQUENCEVERIFY`/`OP_NOP3`.  The fourth case is many-to-many in which a list of opcodes map to a list of terms.  This is used for `OP_2`-`OP_16` as well as the noops (`OP_NOP1`, `OP_NOP4`-`OP_NOP10`).  

There is no case where the number of terms is not equal to the number of opcodes and the number of opcodes is not 1.

Reply:
cool, sounds good, then